### ISSUE TITLE
snf.compute.group_predict Return Array Size

### ISSUE THREAD

**Issue by cmhardage** [rmarkello/snfpy#14]
I am trying to use snf.compute.group_predict. If I reading the documentation correctly, I should be getting an np.ndarray of the first dimension of my test array. Instead, I appear to be getting an np.ndarray the size of the first dimension of my training array. My training data has 4000 samples in three data types with 109 features each. My test data has 1000 samples in three data types with 109 features each. The predicted labels returned is an array of 4000 elements. 

When I examine the code for snf.compute.group_predict, I see:
```
predicted_labels = groups[propogated_labels[len(test[0]):].argmax(axis=1)]
```
If I am reading this correctly, the code starts pulling data from the propogated_labels array after the length of the test set and goes to the end, which would extract the same number of elements as the size of the training set. Is this the expected behavior?

**Comment by rmarkello** [comment]
Hey @cmhardage ! Thanks for the issue :smile: 

This is definitely not the expected behavior, no! It seems like replacing `test[0]` with `train[0]` should do the trick in pulling the correct labels, though. Hopefully a one-line fix is all that's needed!

Would you be interested in contributing a pull request to fix the issue? Let me know, and thanks again for bringing this up.
